tests/external_executor/executors/util.py

```diff
@@ -33,8 +53,78 @@ def terminate(self):
 
 
 @contextmanager
-def executor_thread(executor):
+def executor_thread(executor: Executor):
     et = ExecutorThread(executor)
     et.start()
     yield executor
     et.terminate()
+
+
+class ExecutorContainer:
+    def __init__(
```

Comment:
Let's not have files called util, and mix code meant to be used inside the executor, and outside to orchestrate it, in the same file.

Reply:
Moved the code around in https://github.com/microsoft/CCF/pull/5018/commits/682a90dec94d35e41d8a4a28d63755c89593367b and https://github.com/microsoft/CCF/pull/5018/commits/ad50472969d403ae2a4150e8e61cfd05803a66fb